lib/addMethod/globalize/before.js

```diff
@@ -5,36 +5,55 @@
 * object - need to ensure that variables are passed and saved correctly in the
 * tests pre substitution.
 */
-var when = require('when');
-var _    = require('lodash');
+const when = require('when');
+const _    = require('lodash');
+
+const localOnly = require('./localOnly');
+
+const REFERENCE_MODIFICATION_ERROR_MESSAGE = 'Modification by reference is deprecated. `before` must return the modified object.';
+
+function validateResult (referencedParams, originalParamsCopy) {
```

Comment:
this function is very similar to the one in beforeRequest, it could be put in its own file and reused.